Makefile

```
@@ -33,11 +38,11 @@ preview: docs-preview ## Alias for docs-preview
 
 .PHONY: broken-links
 broken-links: ## Run broken links check
-	docker run --rm -it \
-		--name $(CONTAINER_NAME) \
-		-p $(PORT):$(PORT) \
-		-v $(PWD):/docs \
-		$(MINT_IMAGE) broken-links
+	docker run --rm $(DOCKER_TTY) \
+	  --name $(CONTAINER_NAME) \
+	  -p $(PORT):$(PORT) \
+	  -v $(PWD)/public:/docs \
+	  $(MINT_IMAGE) broken-links
```

```suggestion
	docker run --rm -t \
	  -v $(PWD)/public:/docs \
	  $(MINT_IMAGE) broken-links
```
We shouldn't force a name for this container or bind ports as it is discarded after it's run.

Is there a reason to break `-it` into a variable only for github? We don't need `-i` but I _think_ we'd at least want `-t` so the messages are printed to stdout